### PR TITLE
amp flags --version, -V function correctly

### PR DESCRIPTION
**Pull request by neha-viswanathan**
Fix for issues #547 #561 #565 

* Fixed `amp --version`, `amp -V` to only display amp version
* Formatted help descriptions
* Formatted amplifier flags to lowercase

To test #547 
```
$ make install
$ amp --version
amp (cli version: v0.4.0-dev, build: bdc1a369)
$ amp -V
amp (cli version: v0.4.0-dev, build: bdc1a369)
```

### CODE REVIEW

**Review by JosephGJ — APPROVED** [review]
LGTM

File: cmd/amp/platform_manager.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func (s *ampManager) init(firstMessage string) error {
 	defaultHeaders := map[string]string{"User-Agent": "amplifier"}
 	cli, err := client.NewClient(DockerURL, DockerVersion, nil, defaultHeaders)
 	if err != nil {
-		return fmt.Errorf("impossible to connect to Docker on: %s\n%v", DockerURL, err)
+		return fmt.Errorf("Impossible to connect to Docker on: %s\n%v", DockerURL, err)
```
Comment (ndegory) [inline cmd/amp/platform_manager.go]: Error message [should not be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings). A PR is waiting to be merged with same kind of warnings: #545

File: cmd/amp/registry.go [hunk 2]
```diff
@@ -94,7 +94,7 @@ func RegistryPush(amp *client.AMP, cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Error parsing reference: %q is not a valid repository/tag", image)
 	}
 	if _, isCanonical := distributionRef.(distreference.Canonical); isCanonical {
-		return errors.New("refusing to create a tag with a digest reference")
+		return errors.New("Refusing to create a tag with a digest reference")
```
Comment (ndegory) [inline cmd/amp/registry.go]: should not be capitalized

File: cmd/amp/topic-create.go [hunk 3]
```diff
@@ -27,11 +27,11 @@ func init() {
 
 func createTopic(amp *client.AMP, cmd *cobra.Command, args []string) error {
 	if len(args) == 0 {
-		return errors.New("must specify topic name")
+		return errors.New("Must specify topic name")
```
Comment (ndegory) [inline cmd/amp/topic-create.go]: should not be capitalized

File: cmd/amp/topic-create.go [hunk 4]
```diff
@@ -27,11 +27,11 @@ func init() {
...
 	}
 	name := args[0]
 	if name == "" {
-		return errors.New("must specify topic name")
+		return errors.New("Must specify topic name")
```
Comment (ndegory) [inline cmd/amp/topic-create.go]: ditto

**Review by ndegory — CHANGES_REQUESTED** [review]
LGTM, with requested changes. Also, we should probably add a Long Cobra description to the RootCmd, but that means changing the usage template in main.go.

**Review by subfuzion — CHANGES_REQUESTED** [review]
@neha-viswanathan I agree with @ndegory. Looks good overall. Please make the requested changes and also resolve merge conflicts. Thanks.

**Comment by neha-viswanathan** [comment]
@ndegory I will add the Cobra Description to the root command as a different PR.

**Review by ndegory — APPROVED** [review]

**Review by subfuzion — APPROVED** [review]

**Comment by ndegory** [comment]
@neha-viswanathan there's an issue with this PR (already merged), previous commits (at least a part of #592) have been reverted. I noticed it with the output of ```amp config```.
I think you should revert it, restore the branch, and rebase it from master.

**Pull request merged**